README.md

```diff
@@ -1 +1,46 @@
 # curriculum-cr-training-mobile-menu
+
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+## Built With
+
+- Html5 and Css3
+- Javascript
+
+### Setup
```

Comment:
- [ ] You have completed most of the last requests, but there is a little thing. As given in the [template](https://github.com/microverseinc/readme-template) above, you also need to have a `Live Demo` link to your project. Kindly recommend you fix this issue.